Git clone on windows Ant fails to build. "encoding Cp1252"

Cloned a copy 4 aug 2016 Windows as per wiki ant fails on Webo.
Will try again to download zip.
Confirmed same error with zip file.

Used to work fine I wrote [this](https://github.com/loklak/loklak_server/wiki/Setting-up-Loklak-on-Windows-32-Bit)
Also yacy is a service now.

Command prompt output.
init:

build:
   [delete] Deleting directory f:\loklak_server\classes
    [mkdir] Created dir: f:\loklak_server\classes
     [echo] loklak: f:\loklak_server\build.xml
    [javac] Compiling 196 source files to f:\loklak_server\classes
    [javac] f:\loklak_server\src\org\loklak\api\search\WeiboUserInfo.java:88: er
ror: unmappable character for encoding Cp1252
    [javac]                                     case "µÇºÕ?ûÕ?æ´?Ü":
    [javac]                                               ^
    [javac] f:\loklak_server\src\org\loklak\api\search\WeiboUserInfo.java:88: er
ror: unmappable character for encoding Cp1252
    [javac]                                     case "µÇºÕ?ûÕ?æ´?Ü":
    [javac]                                                  ^
    [javac] 2 errors

BUILD FAILED
f:\loklak_server\build.xml:99: Compile failed; see the compiler error output for
 details.

Total time: 3 seconds

f:\loklak_server>

xml output file
<?xml version="1.0" encoding="UTF-8" standalone="no"?>
<project basedir="." default="build" name="loklak">
    <property environment="env"/>
    <property name="target" value="1.8"/>
    <property name="source" value="1.8"/>
    <path id="loklak.classpath">
        <pathelement location="classes"/>
        <pathelement location="lib/antlr-runtime-3.5.jar"/>
        <pathelement location="lib/asm-4.1.jar"/>
        <pathelement location="lib/asm-commons-4.1.jar"/>
        <pathelement location="lib/bcprov-jdk15on-154.jar"/>
        <pathelement location="lib/bcpkix-jdk15on-154.jar"/>
        <pathelement location="lib/commons-cli-1.3.1.jar"/>
        <pathelement location="lib/commons-logging-1.2.jar"/>
        <pathelement location="lib/compiler-0.8.13.jar"/>
        <pathelement location="lib/compress-lzf-1.0.2.jar"/>
        <pathelement location="lib/elasticsearch-2.3.2.jar"/>
        <pathelement location="lib/groovy-all-2.4.4.jar"/>
        <pathelement location="lib/guava-18.0.jar"/>
        <pathelement location="lib/hamcrest-core-1.3.jar"/>
        <pathelement location="lib/HdrHistogram-2.1.6.jar"/>
        <pathelement location="lib/hppc-0.7.1.jar"/>
        <pathelement location="lib/httpclient-4.5.2.jar"/>
        <pathelement location="lib/httpcore-4.4.4.jar"/>
        <pathelement location="lib/httpmime-4.5.2.jar"/>
        <pathelement location="lib/jackson-core-2.6.2.jar"/>
        <pathelement location="lib/jackson-dataformat-cbor-2.6.2.jar"/>
        <pathelement location="lib/jackson-dataformat-smile-2.6.2.jar"/>
        <pathelement location="lib/jackson-dataformat-yaml-2.6.2.jar"/>
        <pathelement location="lib/javax.servlet-api-3.1.0.jar"/>
        <pathelement location="lib/javax.mail-1.5.5.jar"/>
        <pathelement location="lib/jcommon-1.0.18.jar"/>
        <pathelement location="lib/jetty-http-9.3.9.v20160517.jar"/>
        <pathelement location="lib/jetty-io-9.3.9.v20160517.jar"/>
        <pathelement location="lib/jetty-rewrite-9.3.9.v20160517.jar"/>
        <pathelement location="lib/jetty-security-9.3.9.v20160517.jar"/>
        <pathelement location="lib/jetty-server-9.3.9.v20160517.jar"/>
        <pathelement location="lib/jetty-servlet-9.3.9.v20160517.jar"/>
        <pathelement location="lib/jetty-servlets-9.3.9.v20160517.jar"/>
        <pathelement location="lib/jetty-util-9.3.9.v20160517.jar"/>
        <pathelement location="lib/jetty-webapp-9.3.9.v20160517.jar"/>
        <pathelement location="lib/jfreechart-1.0.19.jar"/>
        <pathelement location="lib/jna-4.1.0.jar"/>
        <pathelement location="lib/joda-convert-1.2.jar"/>
        <pathelement location="lib/joda-time-2.8.2.jar"/>
        <pathelement location="lib/json-schema-validator-2.2.6-lib.jar"/>
        <pathelement location="lib/jsoup-1.9.2.jar"/>
        <pathelement location="lib/jsr166e-1.1.0.jar"/>
        <pathelement location="lib/jts-1.13.jar"/>
        <pathelement location="lib/junit-4.12.jar"/>
        <pathelement location="lib/log4j-1.2-api-2.6.jar"/>
        <pathelement location="lib/log4j-api-2.6.jar"/>
        <pathelement location="lib/log4j-core-2.6.jar"/>
        <pathelement location="lib/log4j-slf4j-impl-2.6.jar"/>
        <pathelement location="lib/lucene-analyzers-common-5.5.0.jar"/>
        <pathelement location="lib/lucene-backward-codecs-5.5.0.jar"/>
        <pathelement location="lib/lucene-core-5.5.0.jar"/>
        <pathelement location="lib/lucene-grouping-5.5.0.jar"/>
        <pathelement location="lib/lucene-highlighter-5.5.0.jar"/>
        <pathelement location="lib/lucene-join-5.5.0.jar"/>
        <pathelement location="lib/lucene-memory-5.5.0.jar"/>
        <pathelement location="lib/lucene-misc-5.5.0.jar"/>
        <pathelement location="lib/lucene-queries-5.5.0.jar"/>
        <pathelement location="lib/lucene-queryparser-5.5.0.jar"/>
        <pathelement location="lib/lucene-sandbox-5.5.0.jar"/>
        <pathelement location="lib/lucene-spatial-5.5.0.jar"/>
        <pathelement location="lib/lucene-spatial3d-5.5.0.jar"/>
        <pathelement location="lib/lucene-suggest-5.5.0.jar"/>
        <pathelement location="lib/netty-3.10.5.Final.jar"/>
        <pathelement location="lib/securesm-1.0.jar"/>
        <pathelement location="lib/snakeyaml-1.15.jar"/>
        <pathelement location="lib/slf4j-api-1.7.21.jar"/>
        <pathelement location="lib/spatial4j-0.5.jar"/>
        <pathelement location="lib/t-digest-3.0.jar"/>
        <pathelement location="lib/twitter4j-core-4.0.2.jar"/>
        <pathelement location="lib/twitter4j-stream-4.0.2.jar"/>
    	<pathelement location="lib/jdom-1.1.3.jar"/>
    	<pathelement location="lib/rome-1.0.jar"/>
    </path>
    
    <target name="init">
        <copy includeemptydirs="false" todir="classes">
            <fileset dir="src">
                <exclude name="**/*.launch"/>
                <exclude name="**/*.java"/>
            </fileset>
        </copy>
    </target>
    
    <target name="clean">
        <delete dir="classes"/>
        <delete dir="html/javadoc"/>
    </target>
    
    <target depends="init" name="build">
        <delete dir="classes"/>
        <mkdir dir="classes"/>
        <echo message="${ant.project.name}: ${ant.file}"/>
        <javac debug="true" destdir="classes" includeantruntime="false" source="${source}" target="${target}">
            <src path="src"/>
            <classpath refid="loklak.classpath"/>
        </javac>
    </target>
    
    <target name="javadoc" depends="init" description="make javadoc">
        <delete dir="html/javadoc"/>
        <javadoc destdir="html/javadoc" windowtitle="loklak javadoc" encoding="UTF-8" charset="UTF-8" access="private">
            <classpath refid="loklak.classpath"/>
            <fileset dir="src">
                <include name="**/*.java"/>
            </fileset>
        </javadoc>
    </target>
    
    <target depends="build,javadoc" name="all"/>
    
    <target name="start">
        <java classname="org.loklak.Main" failonerror="true" fork="yes">
            <jvmarg line="-ea"/>
            <classpath refid="loklak.classpath"/>
        </java>
    </target>
    
    <target name="jar" depends="build">
        <mkdir dir="dist"/>
	    <manifestclasspath property="jar.classpath" jarfile="dist/loklak.jar">
		    <classpath refid="loklak.classpath" />
	    </manifestclasspath>
	    <jar destfile="dist/loklak.jar" basedir="classes/">
		    <manifest>
			    <attribute name="Class-Path" value="${jar.classpath}" />
			    <attribute name="Main-Class" value="org.loklak.LoklakServer" />
		    </manifest>
	    </jar>
    </target>
    
</project>

--- a/build.xml
+++ b/build.xml
@@ -28,7 +28,7 @@
         <delete dir="classes"/>
         <mkdir dir="classes"/>
         <echo message="${ant.project.name}: ${ant.file}"/>
-        <javac debug="true" destdir="classes" includeantruntime="false" source="${source}" target="${target}">
+        <javac debug="true" destdir="classes" includeantruntime="false" source="${source}" target="${target}" encoding="UTF-8">
             <src path="src"/>
             <classpath refid="loklak.classpath"/>
         </javac>

